### ISSUE TITLE
Allow to run without installing globally with npm

### ISSUE THREAD

**Issue by dsebastien** [devspace-cloud/devspace#790]
**Is your feature request related to a problem?**  
No

**Which solution do you suggest?**  
Allow adding devspace as a devDependency in package.json and making sure that it does not depend on a global installation.

For consistency across developer machines, we prefer to add tools in the devDependencies of our projects. Unfortunately it seems that devspace only works if it gets installed globally using npm install -g devspace. 

**Additional context**  

If I add devspace as a devDependency then try to execute it through an npm script, I get the following error:
```
...
> devspace sync

Finishing installation for DevSpace CLI
internal/modules/cjs/loader.js:626
    throw err;
    ^

Error: Cannot find module '/home/sebastien/.config/yarn/global/node_modules/devspace/index.js'
```

As it stands the package only works when installed globally.

/kind feature

**Comment by LukasGentele** [comment]
Thanks for opening this issue, @dsebastien. Since the latest release, this should actually work with npm as well as yarn. Could you check the version that is being installed after you run `npm install` or `yarn` using either:
```bash
npm list devspace
```
or 
```bash
yarn list --pattern devspace
```

If the version is v4.2.0 or lower without `-npm` suffix, please do this:

1. Uninstall the package using either `npm uninstall --save devspace` or `yarn remove devspace`
2. Add it again using the patched npm version `4.2.0-npm` by running either `npm install --save-dev devspace@4.2.0-npm` or `yarn add devspace@4.2.0-npm --dev`

From v4.2.1 on which will be released very soon, you will not need the `-npm` suffix anymore.

**If you are already using `v4.2.0-dev`, let me know. Then, there might be a bug in the npm/yarn installer.**

**Comment by dsebastien** [comment]
Works great with the `-npm` suffix!
Awesome, thanks a lot for your help :)
